docs/compilation.rst

```diff
@@ -26,7 +25,9 @@ Prerequisites
   |                                                                       |                                                                                        |
   |                                                                       | Required for building DALI TensorFlow Plugin.                                          |
   +-----------------------------------------------------------------------+----------------------------------------------------------------------------------------+
-
+  | `Git LFS <https://git-lfs.com/>`_                                     | Follow installation manual appropriate for your operating system.                      |
```

Comment:
I wonder if we shouldn't state that after cloning repo and subrepos, and installing Git LFS the user needs to update the subrepos once again to pull the binary blobs.

Reply:
Agree, added the note.